internal/graphql/graphql.go

```diff
@@ -17,7 +19,40 @@ type Error struct {
 	Message string `json:"message"`
 }
 
-func ExecuteGraphQL(gql string, config *target.BaseTargetConfig) ([]byte, error) {
+type dummyResultObject struct{}
+
+func ExecuteGraphQLWithoutResponse(gql string, config *target.BaseTargetConfig) error {
+	result := dummyResultObject{}
```

Comment:
I think you can just do 'struct{}{}' here so you don't need to define a custom struct type